### ISSUE TITLE
Is `crszonev4a` actually an US set?

### ISSUE THREAD

**Issue by FMecha** [mamedev/mame#12974]
### MAME version

0.271 (mame0271-102-g3cc7f278ddb)

### System information

Windows 10 22H2 [Version 10.0.19045.5011] 64-bit, English
Acer Aspire A315-55G (Intel Core i5-1021U / NVIDIA GeForce MX230 / 16 GB RAM)
Default settings for most part.

### Emulated system/software

crszonev4a

### Incorrect behaviour

`crszonev4a` is supposedly a "(World, CSZO4 Ver. A)" version. However, when it boots, there is the presence of AAMA rating screen, as well as higher default coin setup (4 to start/2 to continue), which is more in line of an US set. For comparison, the parent `crszone` set, labeled "(World, CSZO4 Ver. B)", has 2 to start/1 to continue coinage and no AAMA rating screen (but has green hit sparks; crszonev2a/v2b, the World/CZO2 versions, doesn't have green hit sparks).

[FIGHTERS CHOICE Arcade on YouTube](https://youtu.be/acxoAeUGcoQ) claims that their Crisis Zone board with green hit sparks is CZO4 Ver. A, which makes me wonder if `crszonev4a` might be mislabeled. (The current "CZO4 Ver. A" in MAME has orange hit sparks.)

### Expected behaviour

`crszonev4a` might need to be re-labeled as an US set. This would mean that it would be an alternate version of CZO3, given Namco region numbering:

- 1: Japan
- 2: World
- 3: US
- 4: World (alt) 
- 5: US (alt) - according to Time Crisis II (`timecrs2v5a`) anyway.

### Steps to reproduce

Boot `crszonev4a`, and note the AAMA rating screen on startup.

**Comment by mamehaze** [comment]
yeah, as per @Hydreigon223 the Namco numbering isn't tied to region, there are quite a few examples that prove as much.

Most likely just the order they go into production, which is usually the same simply due to market conditions, but not always.

You have to look at the evidence based on the sets, and yeah, the US warning screens being on one set, but not another is a pretty good indicator.  

Reminds me, Truck Kyosokyoku (Japan, TKK2/VER.A) is definitely NOT a Japanese version either, as the Japanese language option isn't even available in that set (it's in the code, but you don't even have the ability to enable it)   Despite the very Japanese presentation and title screen that one stands quite a high chance of actually being a rare US set, or maybe Asia (it has the parental advisory warning)

**Comment by FMecha** [comment]
Seems like this might be addressed with PR #13994 - requesting this be marked as solved when that gets merged.
